### DISCUSSION TITLE
“Double Commander.app” can’t be opened because Apple cannot check it for malicious software

### DISCUSSION THREAD

**Discussion by andherson17** [doublecmd/doublecmd#1822] [General]
I'm a 1st time Mac owner and was excited to find a file manager that function much like FreeCommander in Windows. But I tried installing it to my Mac Mini M1 and got this message when I tried to open it. Is this a known issue. Do other Mac owners have this issue?

**Comment by alexx2000** [comment]
#1381

**Reply by andherson17** [reply]
Excellent! Thanks.
